joint_limits_interface/test/joint_limits_interface_test.cpp

```diff
@@ -0,0 +1,615 @@
+// Copyright 2013, PAL Robotics S.L. All rights reserved.
+// All rights reserved.
+//
+// Software License Agreement (BSD License 2.0)
```

Comment:
Is this potential issue in the `ros2_control` repository? Currently, we have everything Apache2.0 licensed. If so, can we discuss this with PAL to change the license?

Reply:
If it is a problem we have no objections to change it to Apache2.0

Reply:
@bmagyar